roadmap.md

```diff
@@ -12,6 +12,23 @@ This includes the creation of end user apps, new platform functions or target im
 
 ## Milestones
 
-The PELUX project targets milestone releases twice per year. Our first target is October 2017.
+The PELUX project targets milestone releases twice per year. Our first target is September 2017.
```

Comment:
Just realized that it might be nice to write: 

"The PELUX project targets to provide biannual releases. The first target being September 2017."